examples/align/rs-align.cpp

```diff
@@ -31,8 +31,36 @@ enum class direction
 // Forward definition of UI rendering, implemented below
 void render_slider(rect location, float* alpha, direction* dir);
 
+bool check_rgb_is_supported()
+{
+    bool found_rgb = false;
+    rs2::context ctx;
+    for (auto dev : ctx.query_devices())
```

Comment:
Replace with `ctx.query_devices(RS2_PRODUCT_LINE_DEPTH);` to avoid proceeding on Webcam